tensorflow-core/tensorflow-core-api/src/main/java/org/tensorflow/Session.java

```diff
@@ -221,10 +222,8 @@ public Runner fetch(String operation) {
      * @return this session runner
      */
     public Runner fetch(String operation, int index) {
-      Operation op = operationByName(operation);
-      if (op != null) {
-        outputs.add(op.output(index));
-      }
+      Operation op = graph.operationOrError(operation);
```

Comment:
This will break current behavior by throwing an error where it was silently failing before... but I think it's for the best, so let's keep it that way

Reply:
It won't actually change things, I just copied the Session function to Graph.  The null check there was extraneous.  But I agree about the behavior, I'll note it in the docs.